src/test/resources/com/puppycrawl/tools/checkstyle/grammar/InputJava7MultiCatch.java

```diff
@@ -23,7 +23,7 @@ public static void main(String[] args) {
         try {
             FileInputStream in = new FileInputStream("InputJava7MultiCatch.java");
             throw new CustomException();
-        } catch (final FileNotFoundException | CustomException | com.puppycrawl.tools.checkstyle.grammar.InputJava7MultiCatch.AnotherCustomException e) {
+        } catch (final FileNotFoundException | CustomException | AnotherCustomException e) {
```

Comment:
We cannot just delete package name from each class. It's required to test that a check supports fully-qualified names.

In this exact case probably it's better to move this class to new line

Reply:
Alright, will do this

Reply:
Done